packages/artifacts/src/projects.ts

```diff
@@ -0,0 +1,8 @@
+export enum Project {
```

Comment:
used to be a default export
see my comment in PR desc:
> I usually don't like default exports. Because when one installs a lib, there is no way I can know in advance whether the lib has a default export of not... Until I try and see it doesn't work... So I prefer: either only a single default export or only named exports, no mixing of default and named exports.

Reply:
I read it! It makes sense 👍🏽